### PR TITLE
Nuked custom PDF Oklab colorspace code

### PR DESCRIPTION
**Pull request by Dherse**
Not gonna lie, this one makes me a bit sad 😭 

Changes:

- Removes Oklab export script (the last PostScript code in Typst)
- Removes Oklab `DeviceN` color
- Makes Oklab use sRGB in PDF export
- Adds additional stops (32) for Oklab export in PDF (adjusted note on file size to match)

This should solve all outstanding issues in various PDF readers (part of #3003) and see with @LaurenzV on Discord.

We can hope that a future version of ICC would support it. Actually, ICCv4 seems almost sufficient as we tested with Lau but it has compatibility issues with some readers

### CODE REVIEW

**Comment by Dherse** [comment]
Tested in most common PDF readers by @LaurenzV 
typst-0

File: crates/typst/src/visualize/color.rs [hunk 1]
```diff
@@ -1840,6 +1840,11 @@ impl ColorSpace {
             _ => None,
         }
     }
+
+    /// Returns whether this color space is part of the Oklab family.
+    pub fn is_oklab_family(&self) -> bool {
```
Comment (laurmaedje) [inline crates/typst/src/visualize/color.rs:1845]: is this used?

Comment (Dherse) [reply crates/typst/src/visualize/color.rs:1845]: Nope, it was used to add more intermediary stops, but 32 seems to be enough in most cases.

**Comment by laurmaedje** [comment]
Talked to @reknih. He was very happy because it brings us closer to PDF/A. Thank you!

**Pull request merged**